spring-cloud-task-batch/src/main/java/org/springframework/cloud/task/batch/configuration/TaskBatchAutoConfiguration.java

```diff
@@ -66,4 +74,31 @@ else if(CollectionUtils.isEmpty(dataSources)) {
 			}
 		}
 	}
+
+	@Configuration
+	@ConditionalOnClass(name = "org.springframework.cloud.deployer.resource.maven.MavenResourceLoader")
+	public static class MavenResourceConfiguration {
+		@Bean
+		public MavenResourceLoader mavenResourceLoader(MavenProperties properties) {
+			return new MavenResourceLoader(properties);
+		}
+
+		@Bean
+		@ConditionalOnMissingBean(DelegatingResourceLoader.class)
```

Comment:
if this config is not included (the condition above is not satisfied), then wouldn't there need to be a `DelegatingResourceLoader` that does not include the `MavenResourceLoader` - since I believe that is expected as an autowired property?